### PR TITLE
Adding tree-sitter-php

### PR DESCRIPTION
**Pull request by aryx**

### CODE REVIEW

**Comment by aryx** [comment]
cc @Sjord

File: lang/php/projects.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Git URLs of publicly-accessible projects to be used for parsing stats,
+# one per line.
+#
```
Comment (aryx) [inline lang/php/projects.txt:3]: @mjambon could you polulate this? I tried the script but it has asking for a github API key.

**Comment by aryx** [comment]
This will help https://github.com/returntocorp/semgrep/issues/3576

File: lang/php/projects.txt (hunk 1 above)
Comment (aryx) [reply lang/php/projects.txt:3]: actually I just copy-pasted the one we had in semgrep/parsing-stats/php/

Comment (mjambon) [reply lang/php/projects.txt:3]: yeah you have to generate an api key for yourself, preferably with just the read permission so you can copy-paste it in your shell without worrying.

**Pull request merged**